ga4gh_tool_registry/validate.py

```diff
@@ -7,41 +7,20 @@
 
 from . import swg2salad
 import ruamel.yaml as yaml
+import warnings as warnings
 import requests
 from schema_salad.schema import load_schema, validate_doc
 from schema_salad import jsonld_context
 from schema_salad.main import printrdf
+from schema_salad.sourceline import cmap
 from schema_salad.ref_resolver import Loader
+from ruamel.yaml.comments import CommentedSeq, CommentedMap
```

Comment:
Is this used anywhere?

Reply:
https://github.com/ga4gh/tool-registry-validator/pull/1/files#diff-9a53250075f7b337251e785447ce0c7fR47 I think